src/Propel/Generator/Model/Table.php

```diff
@@ -791,7 +791,7 @@ public function setupReferrers($throwErrors = false)
                 $foreignPrimaryKeys = $foreignTable->getPrimaryKey();
                 // check all keys are referenced in foreign key
                 foreach ($foreignPrimaryKeys as $foreignPrimaryKey) {
-                    if (!$foreignPrimaryKey->hasReferrer($foreignKey) && $throwErrors) {
+                    if (!$foreignPrimaryKey->hasReferrer($foreignKey) && !$foreignKey->isSkipSql() && $throwErrors) {
```

Comment:
Propel used to allow flexible models tailored for application's needs. This checking for foreign keys merely ties hands. For example, if referent table has composite key, it appears to be impossible to join it by a column outside the key.
Please also consider fully justified and common scenario: i18n tables having primary key(id, locale). Of course in many cases other tables would refer to i18n tables only by id. Another similar common scenario is versioning of entities. 
Maybe it's ok to be strict and thus guarantee scheme validity between different RDBMS and to guide amateurs.
But  [model only relationships](http://propelorm.org/documentation/04-relationships.html#model-only-relationships) feature shouldn't be broken. It doesn't work now as described in docs and my pull request fixes this regression.